rdr_service/dao/message_broker_dao.py

```diff
@@ -0,0 +1,94 @@
+from rdr_service import clock
+from dateutil.parser import parse
+from werkzeug.exceptions import BadRequest
+from rdr_service.dao.base_dao import BaseDao
+from rdr_service.dao.participant_dao import ParticipantDao
+from rdr_service.model.message_broker import MessageBrokerRecord, MessageBrokerMetadata
+
+
+class MessageBrokerDao(BaseDao):
+    def __init__(self):
+        self.message_metadata_dao = MessageBrokerMetadataDao()
+        super(MessageBrokerDao, self).__init__(MessageBrokerRecord)
```

Comment:
:nit Can we call super here first since we are not doing any validations/calcs needed before we are calling `super`.

Reply:
Done